src/textual/app.py

```diff
@@ -3008,11 +3006,16 @@ async def _broker_event(
             return False
         else:
             event.stop()
-        if isinstance(action, (str, tuple)):
+        if isinstance(action, str) or (isinstance(action, tuple) and len(action) == 2):
             await self.run_action(action, default_namespace=default_namespace)  # type: ignore[arg-type]
         elif callable(action):
             await action()
         else:
+            if isinstance(action, tuple) and self.debug:
+                # It's a tuple and made it this far, which means it'll be a
+                # malformed action. This is a no-op, but let's log that
+                # anyway.
+                log.warning(f"{event_name} event has an empty a action!")
```

Comment:
This could be more helpful. It will say something like "click event has an empty action", which might be quite mysterious in the logs.

```python
Can't parse action from {action!r}; check formatting of links.
```

Reply:
Perhaps I've misunderstood this, and something akin to this was my initial take, but it'll always be `()`, which won't really relate to what the dev wrote in their code, and which could end up being more confusing than helpful (they may go looking for `click=foo()` when the issue it's it's just `click=`, etc).

Reply:
You're right, it should really be the raw string pulled from the meta.

Reply:
Digging deeper into this, I don't see any "raw string" in the meta; all I see is an empty tuple in each breaking situation. For example, for all of:

- `[@click]`
- `[@click=]`
- `[@click=()]`

the meta is `{"@click": ()}`.

Moreover, I can't see to come up with an action that can't be parsed that doesn't throw an error right away (way before the application has got into a state where user input is permitted). Perhaps I'm missing the obvious here but this seems to be very much about capturing (and ignoring) an empty action, and logging it.

Reply:
Ok, but lets attempt a better error message. Suggest "Can't parse @click action from style meta; check your console markup syntax."